kolibri/plugins/coach/assets/src/views/exams/CreateExamPage/index.vue

```diff
@@ -32,12 +33,11 @@
       </UiAlert>
 
       <div class="buttons-container">
-        <KButton :text="$tr('preview')" @click="preview" />
         <KButton
-          :text="$tr('saveButtonlabel')"
+          :text="$tr('continueButtonlabel')"
```

Comment:
I tend towards these naming schemes because of some feedback I got from @carinediaz - that the context helps translators. @radinamatic have any ideas about this?